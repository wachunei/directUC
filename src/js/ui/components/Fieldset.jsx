import styled from "styled-components";

const Fieldset = styled.fieldset`
  margin: 0 0 1rem 0;
  border: 0;
  padding: 0;
`;

export default Fieldset;
