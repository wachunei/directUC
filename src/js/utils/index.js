export const removeSuffix = (suffix, value) =>
  value.endsWith(suffix) ? value.slice(0, -suffix.length) : value;

export const parseDistinguishedName = (name) =>
  name.replace(
    /\w\S*/g,
    (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase()
  );

export const isFirefox = !window.chrome.app;
