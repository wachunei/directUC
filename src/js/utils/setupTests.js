// eslint-disable-next-line import/no-extraneous-dependencies
import "jest-styled-components";
