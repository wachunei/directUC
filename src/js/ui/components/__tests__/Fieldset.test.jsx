import React from "react";
import Fieldset from "../Fieldset";
import { renderTree } from "./utils";
import "jest-styled-components";

describe("Fieldset Component", () => {
  test("should match snapshot", () => {
    const tree = renderTree(<Fieldset />);
    expect(tree).toMatchSnapshot();
  });
});
