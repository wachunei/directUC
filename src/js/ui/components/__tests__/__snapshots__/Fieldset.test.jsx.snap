// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fieldset Component should match snapshot 1`] = `
.c0 {
  margin: 0 0 1rem 0;
  border: 0;
  padding: 0;
}

<fieldset
  className="c0"
/>
`;
